env/io_posix.cc

```diff
@@ -825,7 +825,8 @@ Status PosixWritableFile::Close() {
     // but it will be nice to log these errors.
     int dummy __attribute__((__unused__));
     dummy = ftruncate(fd_, filesize_);
-#if defined(ROCKSDB_FALLOCATE_PRESENT) && !defined(TRAVIS)
+#if defined(ROCKSDB_FALLOCATE_PRESENT) && defined(FALLOC_FL_PUNCH_HOLE) && \
```

Comment:
Is there any harm of keeping `defined(FALLOC_FL_PUNCH_HOLE)` here?

Reply:
The `fallocate` call below uses `FALLOC_FL_PUNCH_HOLE`. We need to verify `defined(FALLOC_FL_PUNCH_HOLE)` here before attempting to compile that code.

Let me know if I misunderstood your question.

Reply:
Oh sorry I misunderstood it.